inc/servervars.php

```diff
@@ -80,6 +80,8 @@ function get_major_version($version) {
     return preg_replace('/^(\\d+)\\.(\\d+).*$/', '$1.$2', $version);
   }
 
+  // Uses official API for version checking, but is not optimized for
+  // version-checking against multiple release tiers.
   function get_keymanweb_version($tier) {
```

Comment:
While I 'could' delete the old function, I figure that our concern re: internal API merited preservation of the original method.  That way, someone investigating use of the site's code would have an example of how to perform the version-lookup "correctly" in an officially-supported manner.

If it'd be better to outright-remove, just say the word.